src/mbgl/algorithm/update_renderables.hpp

```diff
@@ -26,6 +26,7 @@ void updateRenderables(GetTileFn getTile,
                        const std::optional<uint8_t>& maxParentOverscaleFactor = std::nullopt) {
     std::unordered_set<OverscaledTileID> checked;
     bool covered;
+    bool parentOrChildTileFound;
```

Comment:
You can just initialize it to false here.

Reply:
It is initialized with false in the right place. There is a for loop.

Reply:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-always